### DISCUSSION TITLE
ui.select text marquee

### DISCUSSION THREAD

**Discussion by AlexYRM** [zauberzeug/nicegui#1019] [Ideas / Feature Requests]
Hello, 

First, I would like to apologize if this is already a feature. I have searched and did not find any idea on how to do it. 

I am using a ui.select and in the list I have a few items which are really long. After I select a certain item, it is so long that only a certain part of it is displayed. I would like to ask if there is any way to marquee to the right or down. 
If it is not a feature, could it be implemented in the future ? 

Thank you! 

![select](https://github.com/zauberzeug/nicegui/assets/125361371/ffa9345b-9ac0-4061-99e3-b58d91d98b22)

**Comment by falkoschindler** [comment]
Hi @AlexYRM, that's an interesting request.
To my knowledge, Quasar does not support a scrolling input text. And the only CSS trickery I quickly came up with looks far from useful:

```py
ui.add_head_html('''
<style>
@keyframes marquee {
  0%   { transform: translateX(100%); }
  100% { transform: translateX(-100%); }
}
.q-field__native {
  overflow: hidden;
  animation: marquee 5s linear infinite;
}
</style>
''')

ui.input(value='A very long text that does not fit into the input field').classes('w-32 animated-input')
```

But to cite ChatGPT:
> It's important to note that the marquee effect, which causes text to scroll across the screen, is considered outdated and is not recommended for modern web development. It's often not accessible or user-friendly, and the <marquee> HTML tag is obsolete and not supported in HTML5.
>
> However, if you want to achieve a similar scrolling effect in CSS, you could use animations. But this isn't typically done on an input tag, as input tags are intended for user input, and having the text move could be frustrating for users. [...]

**Reply by AlexYRM** [reply]
Understood. Thanks for the reply!
